manifests/bootstrap_rpm.pp

```diff
@@ -19,51 +16,49 @@
 
   $katello_rhsm_setup_script = 'katello-rhsm-consumer'
   $katello_rhsm_setup_script_location = "/usr/bin/${katello_rhsm_setup_script}"
+  $candlepin_cert_rpm_alias_filename = 'katello-ca-consumer-latest.noarch.rpm'
 
-  $candlepin_consumer_name = "katello-ca-consumer-${rhsm_hostname}"
-  $candlepin_consumer_summary = "Subscription-manager consumer certificate for Katello instance ${rhsm_hostname}"
-  $candlepin_consumer_description = 'Consumer certificate and post installation script that configures rhsm.'
+  $bootstrap_rpm_name = "katello-ca-consumer-${rhsm_hostname}"
 
   include trusted_ca
   trusted_ca::ca { 'katello_server-host-cert':
     source  => $katello_server_ca_cert,
     require => File[$katello_server_ca_cert],
   }
 
+  package { 'rpmbuild':
+    ensure => installed,
+  }
+
   include apache
-  file { $katello_www_pub_dir:
+  file { $rpm_serve_dir:
```

Comment:
Does not have to be this PR, but @ekohl how do you envision we should drop management of this here in favor of elevating it to other places we manage Apache?

Reply:
I noticed the exact same thing yesterday. Not entirely sure yet. I'm wondering if we will continue to use an RPM to register or fully move to registration templates. That would remove the need entirely and then there's no reason to make the integration cleaner.

Reply:
The RPM will exist for at least 2-4 more releases as we will want to give users time to adapt their host workflows to the new method before removing the old.

Reply:
I also want to fix it, but let's keep it for the next iteration.